generator/sources/zend/Kaltura/Client/ClientBase.php

```diff
@@ -907,4 +907,29 @@ public static function getKalturaNullValue()
 	{
 		return KalturaNull::getInstance();
 	}
+
+
+	public function getKaltura_Client_Exception($errorMsg, $errorCode, $args)
```

Comment:
Please rename the methods to getClientException and getAPIException, same as php 5.3 lib.